.pipelines/int-release.yml

```diff
@@ -34,4 +33,4 @@ stages:
       e2eSubscription: $(e2e-subscription)
       billingE2EPipelineName: $(billing-e2e-pipeline-name)
       billingE2EBranchName: $(billing-e2e-branch-name)
-      fullDeploy: ${{ parameters.fullDeploy }}
+      fullDeploy: "true"
```

Comment:
This got us multiple times in INT. Including yesterday with console serial logs Storage account...